Holovibes/sources/gui/windows/lightui.cc

```diff
@@ -165,6 +163,8 @@ void LightUI::notify()
     ui_->ContrastMaxDoubleSpinBox->setValue(api.contrast.get_contrast_max());
 
     ui_->actionSettings->setEnabled(api.input.get_camera_kind() != CameraKind::NONE);
+
+    ui_->RecordedEyePushButton->setText(QString::fromStdString(API.record.get_recorded_eye_display_string()));
```

Comment:
In the main window you disable the recorded eye box but not in the lightui. IS it intended ?